OpenRA.Game/Settings.cs

```diff
@@ -224,6 +224,8 @@ public class GraphicSettings
 
 		public int BatchSize = 8192;
 		public int SheetSize = 2048;
+		public int FontSheetSize = 512;
+		public int CursorSheetSize = 512;
```

Comment:
These all really need to live in mod.yaml, where they are controlled by the mod, rather than settings.yaml which can only be (but overwhelmingly isn't) controlled by the user!

I had taken an initial stab at reducing the need for `SheetSize` via the `SpriteSequenceLoader`. The remaining uses happen before the manifest info is parsed so its not clear how to handle those - but fonts and cursors are loaded late enough that it shouldn't be a problem.

Reply:
The yak shaving shall commence.

Reply:
Just to be clear: i don't think we should be moving `SheetSize` or `BatchSize` in this PR, just the new two.

Reply:
Moved to Manifest.